### ISSUE TITLE
SponsorBlock Remove not working

### ISSUE THREAD

**Issue by megabugman** [BKSalman/ytdlp-gui#45]
SponsorBlock Remove does not work, video contains the ad after download.
Video Link: https://www.youtube.com/watch?v=FwENtwtY0EI

I've tried another yt-dlp gui and it works there while using the same yt-dlp & ffmpeg version.

**Comment by BKSalman** [comment]
Does Mark work?

**Comment by megabugman** [comment]
> Does Mark work?

No.

**Comment by BKSalman** [comment]
it works for me

**Comment by megabugman** [comment]
I'm on Windows, I guess you are not.

**Comment by BKSalman** [comment]
no, I use Linux, but it would be weird if it was an OS specific bug

**Comment by megabugman** [comment]
https://i.postimg.cc/g0qfH3yX/bug.gif

**Comment by BKSalman** [comment]
is this issue still present?

**Comment by megabugman** [comment]
yes, issue persists.

![Image](https://github.com/user-attachments/assets/16d368ab-aaa4-4fff-ae3c-b252a523f988)
I also get an Postprocessing error

**Comment by BKSalman** [comment]
> yes, issue persists.
> 
> ![Image](https://github.com/user-attachments/assets/16d368ab-aaa4-4fff-ae3c-b252a523f988) I also get an Postprocessing error

are you running `ytdlp-gui.exe` from the same folder where `ffmpeg.exe` and `yt-dlp.exe` exist?

**Comment by megabugman** [comment]
yes, same folder
